arch/wasm32/wasm.js

```diff
@@ -821,66 +825,70 @@ var musl_hack = (function() {
   return {
     abort: NYI('abort'),
     __addtf3: NYI('__addtf3'),
-    __block_all_sigs: NYI('__block_all_sigs'),
+    __clone: NYI('__clone'),
     __divtf3: NYI('__divtf3'),
-    __dl_invalid_handle: NYI('__dl_invalid_handle'),
-    __dl_seterr: NYI('__dl_seterr'),
+    __eqtf2: NYI('__eqtf2'),
+    exit: NYI('exit'),
```

Comment:
Seems like you delete things, aren't they needed anymore?

Reply:
This whole section is auto-generated based on the above command... so its precisely what is needed by musl.wast